esphome/ds18b20_beta.yaml

```
@@ -0,0 +1,36 @@
+  package_import_url: github://scns/myESPhome/esphome/ds18b20_beta.yaml
+
+packages:
+  remote_package:
+    url: https://github.com/scns/myESPhome
+    ref: main
+    files:
+      - esphome/package/ds18b20_update_beta.yaml
+      - esphome/package/basis_esp8266_beta.yaml #specific esp8266 basis settings
+      - esphome/package/ds18b20_beta.yaml
+      - esphome/package/basis_settings_beta.yaml
+    refresh: 0d
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)